nextcord/ext/commands/errors.py

```diff
@@ -859,6 +859,14 @@ class NoEntryPointError(ExtensionError):
     def __init__(self, name: str) -> None:
         super().__init__(f"Extension {name!r} has no 'setup' function.", name=name)
 
+class InvalidSetupArguments(ExtensionError):
```

Comment:
you havent added this to ``docs/ext/commands/api.rst``?

```rst
.. autoexception:: nextcord.ext.commands.InvalidSetupArguments
    :members:
```
then ``.. exception_hierarchy::`` should contain it too (no need to mention "This inherits from :exc:`ExtensionError`" as the exception hierarchy shows this)

Reply:
In relation to
>  (no need to mention This inherits from :exc:`ExtensionError` as the exception hierarchy shows this)
All other errors have it